### PR TITLE
Fix accumulation of results when doing several experiment iterations

### PR DESCRIPTION
**Pull request by GuilhemN**
Hello,

I'm currently extending your code to benchmark new results and optimizations and I noticed that your `experiment.py` file is not aggregating results correctly.

I don't know if you accept contributions here, but anyway I'm sending it so you know :)

### CODE REVIEW

File: measuring/scripts/experiment.py [hunk 1]
```diff
@@ -639,13 +639,15 @@ def main():
...
+            result = ["", "", []]
             fngetter = partial(get_filename,
                 experiment, int_only, rtt_ms, pkt_loss, rate,
             )
             start_time = datetime.datetime.utcnow()
             for _ in range(ITERATIONS):
-                result += experiment_run_timers(experiment, int_only)
+                exp_ret = experiment_run_timers(experiment, int_only)
+                result[0:2] = exp_ret[0:2]
+                result[2] += exp_ret[2]
```
Comment (thomwiggers) [inline measuring/scripts/experiment.py]:
```suggestion
                client_cmd, server_cmd, measurements = experiment_run_timers(experiment, int_only)
                result[0] = client_cmd
                result[1] = server_cmd
                result[2] += measurements
```

This seems slightly clearer, this code is already very, very horrible to comprehend.

**Review by thomwiggers — APPROVED** [review]
Well-spotted, the code is quite horrible...

**Comment by thomwiggers** [comment]
I'm also curious to hear what you're working on :)

**Comment by GuilhemN** [comment]
That's better indeed!

I'm working with @lehugueni, PhD student at EPFL, on optimizing the usage of SIKE in TLS 1.3, we implemented several things like parallelized computations, lazy key computations, and a scheme derived from SIKE to also lazily compute part of the encapsulation (with lowered security assumptions, IND-1CCA).
We're currently running the measurements, we have quite some noise when running parallel experiments though, possibly because we tried to migrate to mininet :) But still the results are promising with a reduction from 53ms handshake to 34ms for our 1cca implem of SIKE-P434-compressed with 0-rtt and 1000 measurements obtained sequentially. 

We already exchanged a few emails with Sofía, we can keep you in the loop as well and exchange on the topic if you'd like :)

**Comment by thomwiggers** [comment]
Neat! Sounds like a nice result, looking forward to the paper :) If there's anything you think I can weigh in on/help with in the mean time don't hesitate to reach out.

**Pull request merged**